tests/tools/ssl_good_cert.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
I'm assuming these these certs are self signed. Can they (keys, cert pems) be generated during test time instead of storing in repo? If these are not self signed, then they likely shouldn't be stored this way.

Reply:
Updating per the offline chat: Let's put a README.md in the ssl cert dir with steps we used to get public key/cert from WPT. Instructions would make it easier for the future maintainer to keep them updated/maintained.